tests/app/helpers.py

```diff
@@ -89,7 +98,7 @@ class JSONUpdateTestMixin(object):
 
     def test_non_json_causes_failure(self):
         response = self.client.open(
-            self.endpoint,
+            'services/1000',
```

Comment:
It didn't work with the 409 coming from the PUT methods. I'm going to revert the 409 change so this should go back.